src/i18n/translations/fr/translation.json

```diff
@@ -44,7 +44,8 @@
       "explore": "Explorer",
       "copiedMsg": "Adresse du portefeuille copiée",
       "walletNotInstalled": "{{wallet}} n'est pas installé",
-      "connectAnotherWallet": "Connecter un autre portefeuille"
+      "connectAnotherWallet": "Connecter un autre portefeuille",
+      "refreshBalances": "Rafraichir les  valeurs"
```

Comment:
Small typo: Fix double empty space. 

But more important, not sure if this is good practise or not @chybisov Wdyt? - usually translations happen on the crowdin level.